lib/RoadizCoreBundle/src/SearchEngine/Subscriber/DefaultNodesSourcesIndexingSubscriber.php

```diff
@@ -4,7 +4,9 @@
 
 namespace RZ\Roadiz\CoreBundle\SearchEngine\Subscriber;
 
+use RZ\Roadiz\CoreBundle\Bag\NodeTypes;
 use RZ\Roadiz\CoreBundle\Entity\NodesSources;
+use RZ\Roadiz\CoreBundle\Entity\NodeType;
```

Comment:
If you refactored the entire class, `RZ\Roadiz\CoreBundle\Entity\NodeType` use statement should be useless.